src/Drivers/Imagick/Modifiers/TextModifier.php

```diff
@@ -31,18 +31,50 @@ public function apply(ImageInterface $image): ImageInterface
         $color = $this->driver()->colorProcessor($image->colorspace())->colorToNative(
             $this->driver()->handleInput($this->font->color())
         );
-
+        
+        $strokeWidth = $this->font->strokeWidth();
+        $strokeColor = $this->driver()->colorProcessor($image->colorspace())->colorToNative(
```

Comment:
This color decoding process is only important if the stroke width is greater than 0 and should therefore only consume resources in this case.